x/tally/types/filters.go

```diff
@@ -164,33 +170,35 @@ func NewFilterStdDev(input []byte, gasCostMultiplier uint64, replicationFactor u
 // max sigma. If less than 2/3 of the reveals are non-outliers, "no
 // consensus" error is returned as well.
 func (f FilterStdDev) ApplyFilter(reveals []RevealBody, errors []bool) ([]bool, bool) {
-	dataList, _ := parseReveals(reveals, f.dataPath, errors)
+	dataList, _ := parseReveals(reveals, f.dataPath, errors, f.maxDataLen)
 	return f.filterFunc(dataList, f.maxSigma, errors, f.replicationFactor)
 }
 
-func detectOutliersSignedInteger[T constraints.Integer](dataList []any, maxSigma Sigma, errors []bool, replicationFactor uint16) ([]bool, bool) {
```

Comment:
I think this method still uses the old approach and not actually the standard deviation?